desci-server/src/services/data/externalUrlProcessing.ts

```diff
@@ -128,10 +128,17 @@ export async function processExternalUrlDataToIpfs({
          */
         if (componentType === ResearchObjectComponentType.PDF) {
```

Comment:
are we sure links will be of type PDF at this point? should the logic at line 134 be moved outside of this check?